platform/windows/detect.py

```diff
@@ -221,7 +221,8 @@ def configure_msvc(env, manual_msvc_config):
 
     LIBS = ['winmm', 'opengl32', 'dsound', 'kernel32', 'ole32', 'oleaut32',
             'user32', 'gdi32', 'IPHLPAPI', 'Shlwapi', 'wsock32', 'Ws2_32',
-	    'shell32', 'advapi32', 'dinput8', 'dxguid', 'imm32', 'bcrypt','Avrt']
+            'shell32', 'advapi32', 'dinput8', 'dxguid', 'imm32', 'bcrypt','Avrt',
```

Comment:
There should be a space before `'Avrt',`. It's not from your commit but it would be worth fixing nevertheless (maybe also put it on the new line with `'dwmapi'` to keep a consistency line wrapping).